float32 x
float32 y
float32 vx
float32 vy
float32 yaw
float32 radius
float32 confidence
int32 tracked_id
string class_name